src/main/java/com/conveyal/analysis/util/MapTile.java

```diff
@@ -1,23 +1,74 @@
 package com.conveyal.analysis.util;
 
+import com.conveyal.gtfs.util.GeometryUtil;
+import com.conveyal.r5.common.GeometryUtils;
+import com.wdtinc.mapbox_vector_tile.adapt.jts.MvtEncoder;
+import com.wdtinc.mapbox_vector_tile.adapt.jts.UserDataKeyValueMapConverter;
+import com.wdtinc.mapbox_vector_tile.adapt.jts.model.JtsLayer;
+import com.wdtinc.mapbox_vector_tile.adapt.jts.model.JtsMvt;
+import com.wdtinc.mapbox_vector_tile.build.MvtLayerParams;
+import org.locationtech.jts.geom.Coordinate;
 import org.locationtech.jts.geom.Envelope;
+import org.locationtech.jts.geom.Geometry;
+import org.locationtech.jts.geom.LineString;
+import org.locationtech.jts.geom.Point;
+import org.locationtech.jts.simplify.DouglasPeuckerSimplifier;
 
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Utilities for creating a slippy map tile and generating it's enveloper. Also for encapsulating it's properties that
```

Comment:
Javadoc should be updated to reflect the fact that this is now vector tile specific, but could eventually be combined with raster tile uses, or be made a subclass of a general tile (not raster or vector specific). It's kind of surprising that we define tile2lat() here and apparently nowhere else in the project - maybe we really don't deal with any raster tiles in the Java code.